docs/ibc/2_IBC_TERMINOLOGY.md

```diff
@@ -0,0 +1,85 @@
+# 2: IBC Terminology
+
+> This is an overview of terms used in IBC specifications.
+
+> For the design rationale behind the protocol, see [here](./1_IBC_DESIGN_RATIONALE.md).
+
+> For an architectural overview, see [here](./3_IBC_ARCHITECTURE.md).
+
+This document provides definitions in plain English of key terms used throughout the IBC specification set.
+
+## Definitions
+
+### Actor
+
+An *actor*, or a *user* (used interchangeably), is an entity interacting with the IBC protocol. An actor can be a human end-user, a module or smart contract running on a blockchain, or an off-chain relayer process capable of signing transactions.
+
+### Chain
+
+A *chain*, or *ledger* (used interchangeably), is a distributed ledger (or "blockchain", although a strict chain of blocks may not be required) implementing part or all of the IBC specification as a component or module within its state machine.
+
+### State Machine
+
+The *state machine* of a particular chain defines the structure of the state as well as the set of rules which determines valid transactions that trigger state-transitions based on the current state agreed upon by the consensus algorithm of the chain.
+
+### Consensus
+
+A *consensus* algorithm is the protocol used by the set of processes operating a distributed ledger to come to agreement on the same state, generally under the presence of a bounded number of Byzantine faults.
+
+### Consensus State
+
+The *consensus state*, or the *root-of-trust*, is the set of information about the state of a consensus algorithm required to verify proofs about the output of that consensus algorithm (e.g. accumulator roots in signed headers).
```

Comment:
Isn't "root of trust" a bit overloaded with a different meaning?

Reply:
I guess it is in Tendermint (at least in the codebase). Should we just use "root of trust" here?